content/get_start/UseCase_Workflows/pollination.md

```diff
@@ -0,0 +1,75 @@
+
+  - [HVAC](https://docs.pollination.cloud/user-manual/rhino-plugin/energy-modeling/assign-hvac-systems): Only "Ideal air loads" option is available by default and hence was selected (which was modeled as zone equipment). However, an HVAC system is a key element to run the PRM measure, thus, HVAC was revisited after exporting the model to OpenStudio. 
+    There is also an option in Pollination to import HVAC systems from OpenStudio that could be considered to assign HVAC systems. 
+
+  - Service Hot Water (SHW): There is an option for SHW but no system to be selected.
+
+  - Space Names: To avoid having random handlers assigned to the space names and thermal zones after the export, "Reset Identifier" option was enabled in Pollination's final step before export. This option typically enables Pollination to parse the correct space names into the OpenStudio model. 
+
+![construction](/BEM-for-PRM/get_start/UseCase_Workflows/images/Constructionsets_programtype_HVAC.png?width=700px&align=right&classes=border,alignCenter)
+
+##### **3. Export the model**
+The model was exported from Pollination by clicking on the export button in the final step (as shown in the image above) as an .osm file. It was further imported to OpenStudio and boundary conditions were checked to make sure it imported correctly. 
+
+|3D User Model in OpenStudio rendered by surface type| 3D User Model in OpenStudio rendered by boundary condition |
+|:-:|:-:|
+|![userdata](/BEM-for-PRM/get_start/UseCase_Workflows/images/Pollination_OpenStudio_Model.png?width=500px&align=right&classes=border,alignCenter)|![userdata](/BEM-for-PRM/get_start/UseCase_Workflows/images/Pollination_Boundary_Condition_model.png?width=500px&align=right&classes=border,alignCenter)|
+
+##### **4. Apply 90.1 PRM method**
+Before applying the 90.1 PRM measure, the exported BEM model was checked for any missing/incorrect information and those were addressed by following the steps below.
+
+- The measure tag for Standard Space Types had to be changed to match the names in the [PRM database](/BEM-for-PRM/user_guide/add_compliance_data/building_type/user_data_building) (Standard Space Types were set to "office - enclosed <= 250 sf", "conference/meeting/multipurpose" and "office - open" depending on the space function). Although this does not cause a simulation error, the PRM baseline measure will not pick the correct space-related input values (e.g., LPD) if this step is skipped.  
+
+- A Packaged Rooftop Air Conditioner, one for each floor was assigned along with the thermal zones. 
+
+**Once the PRM measure was applied on the BEM model, a proposed as well as one to four baseline models following the set of PRM rules were generated.**
```

Comment:
Any images after generated? A proposed model and baseline models.

Reply:
1. I only see a baseline model Figure 7 in the documentation which looked exactly the same as Figure 5 that I used to show the model right after the export, so I thought we do not have images for the Final resulting model. https://confluence.pnnl.gov/confluence/display/OSSTD/Pollination+Revit+Plugin+-+DD+Phase

2. Also, I don't think these workflows were tested for generating proposed models based on what I remember and see in the documentation. 

@AmmarDeh  Can you please confirm both? 

@weilixu Would you also recommend adding the changes that took place after the measure was run, like this? 
The baseline model has a WWR of 30.95% (Figure7-a).

The model shows the correct boundary condition for interior walls and floors (Figure7-b).
New envelope materials (opaque and fenestration) were added by the PRM measure with construction names starting with "PRM...".
The LPD is showing  11.84, 11.84, 13.99 W/m2 for Open Office, Conference and Enclosed Office respectively, which matches [PRM database](https://github.com/NREL/openstudio-standards/blob/master/lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm_2019/data/ashrae_90_1_prm_2019.prm_interior_lighting.json).
The model includes 3 HVAC systems of type Sys5 and 7 systems of type PSZ-AC with a hot water loop.
Slab boundary is "GroudFCfactorMethod".

Reply:
In these test case, the user model is the same as proposed model.
@JayatiChhabra bullet points make sense to me.